CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased] - Requiring major [semver](https://semver.org/) change
+
+### Added
+
+* Support for promises via [request-promise-native](https://github.com/request/request-promise-native), while retaining backwards compatibility for those wishing to use callbacks
+* Ability to add the `apiKey` once... when instantiating the `ZapClient`. See example in [README](https://github.com/zaproxy/zap-api-nodejs/README.md#instantiate-the-client-api)
```

Comment:
Is the ellipsis needed?

Reply:
I thought it read better with a pause. Have removed.